cost/flexera/cco/scheduled_reports/scheduled_report.pt

```diff
@@ -95,6 +104,43 @@ parameter "param_billing_centers" do
   default []
 end
 
+parameter "param_percent_change_threshold" do
+  type "number"
+  category "Policy Settings"
+  label "Percent Change Threshold Value"
+  description "The threshold (in percent) for the percent change in cost between time period(s) that will trigger the report. Default of 0 will send the report every time based on the applied policy schedule."
+  min_value 0
+  max_value 100
+  default 0
+end
+
+parameter "param_percent_change_consecutive_threshold" do
+  type "number"
+  category "Policy Settings"
+  label "Percent Change Consecutive Threshold"
+  description "The number of consecutive time period(s) where there was a percent change in cost between time period(s) that will trigger the report. Default of 0 will send the report every time based on the applied policy schedule."
```

Comment:
Description says default is 0 but default is set to 1, and the min_value would prevent the user from manually setting this to 0.